src/AppInstallerCLICore/Workflows/ConfigurationFlow.cpp

```diff
@@ -1369,27 +1369,31 @@ namespace AppInstaller::CLI::Workflow
         auto getDetailsOperation = configContext.Processor().GetSetDetailsAsync(configContext.Set(), ConfigurationUnitDetailFlags::ReadOnly);
         auto unification = anon::CreateProgressCancellationUnification(std::move(progressScope), getDetailsOperation);
 
+        bool suppressDetailsOutput = context.Args.Contains(Args::Type::ConfigurationAcceptWarning) && context.Args.Contains(Args::Type::ConfigurationSuppressPrologue);

```

Comment:
Should we validate that `supress-prologue` is not used on its own?

Reply:
It only controls output, not the results.  I don't see it as harmful to be passed when it wouldn't do anything.